fdbserver/masterserver.actor.cpp

```diff
@@ -251,6 +251,12 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 
 	Future<Void> logger;
 
+	Reference<EventCacheHolder> masterRecoveryStateHolder;
```

Comment:
Nit: in other places, you've used called these `EventHolder` rather than just `Holder`. It's not a huge deal, but for consistency you could use the same pattern here.